frontend/src/components/Utils/time.js

```diff
@@ -58,3 +60,22 @@ export function timestampToFormatted(msTimestamp, timezone = 'America/New_York')
   let formatted = date.toLocaleString('en-US', formatOptions);
   return formatted;
 }
+
+/**
+ * Returns all the time zones in a country (in pretty format).
+ * 
+ * @param {string} countryName The name of the country for which to get the time zones.
+ * @returns The list of time zones in the provided country.
```

Comment:
What is returned when the catch block is executed? All of the countries? In either case, you may want to specify this here.

Reply:
The list of all the time zones. comment added.